Brocfile.js

```diff
@@ -33,20 +33,26 @@ var es5 = new Babel(lib, {
   blacklist: ['es6.modules']
 });
 
+function rollupConfig(entry) {
+  return new Rollup(es5, {
+    rollup: {
+      entry: 'lib/' + entry,
+      cache: 'BUILD:es6-promise',
```

Comment:
We have two rollups with almost every file repeated so to reuse the same cache (of file reads, AST etc.) for both of them.
https://github.com/rollup/rollup/wiki/JavaScript-API

Let me find a better Doc